### PR TITLE
Proposal 0002: GRC

### PR DESCRIPTION
**Pull request by jaspervdj**
[Rendered Version](https://github.com/haskell-org/committee/blob/proposals/0002-grc.md/proposals/0002-grc.md)

### CODE REVIEW

File: proposals/0002-grc.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+author: Jasper Van der Jeugt
+date-accepted: N/A
+discussion: https://github.com/haskell-org/committee/pull/2
+---
+
+# Guidelines for Respectful Communication
+
+The Haskell.org committee adopts the
+[Guidelines for Respectful Communication](grc).
```
Comment (TikhonJelvis) [inline proposals/0002-grc.md:10]:
```suggestion
[Guidelines for Respectful Communication][grc].
```

Link syntax.

**Review by TikhonJelvis — APPROVED** [review]
Seems like a reasonable starting point. This will align us with other Haskelly organizations, and we can extend/amend our rules as needed in the future.

File: proposals/0002-grc.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+date-accepted: N/A
+discussion: https://github.com/haskell-org/committee/pull/2
+---
+
+# Guidelines for Respectful Communication
+
+The Haskell.org committee adopts the
+[Guidelines for Respectful Communication][grc].
+
+This applies only to members of the board, in in all our public interactions in
```
Comment (emilypi) [inline proposals/0002-grc.md:12]:
```suggestion
This applies only to members of the board, in all our public interactions in
```

File: proposals/0002-grc.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+the Haskell sphere, including email, social media, discussion forums, and so on.
```
Comment (emilypi) [inline proposals/0002-grc.md:13]:
```suggestion
the Haskell sphere, including email, social media, discussion forums, and so on, as they pertain to our status as representatives of Haskell.org.
```

**Review by emilypi — COMMENTED** [review]
Two suggestions, one is a typo.

**Comment by jaspervdj** [comment]
This proposal has been approved; see discussion and votes on the `haskell-community` mailing list [here](https://mail.haskell.org/pipermail/haskell-community/2020-October/000393.html) and [here](https://mail.haskell.org/pipermail/haskell-community/2020-November/000398.html).  I will amend the proposal with these.

**Pull request merged**